### DISCUSSION TITLE
Cohere+/Second Renaissance Ecosystem Mapping Exemplars & Reflections

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#972] [Research]
Below are some examples from our Cohere+/Second Renaissance Ecosystem Mapping (and reflections). Welcoming any comments or questions!

Some reflection prompts:
- What patterns (similarities or differences) do you notice among these examples?
- What other organisations come to mind as you look at these examples and why?
- What tribes/centres/communities might these organisations each be part of?
- What gives you a credible sense that an organisation is really working for (and contributing to) paradigmatic and integrated change?

## 42 Acres

- Profile: https://ecosystem.lifeitself.org/tbd/42-Acres
- Markdown file (metadata fields, such as location, year founded and team size, are visible): https://github.com/life-itself/ecosystem/blob/main/content/tbd/42-Acres.md

![Screenshot 2023-12-20 at 09 11 58](https://github.com/life-itself/community/assets/97066487/9d013aef-399e-4ca9-851f-ad299adead46)

### Keywords

- regenerative/regeneration
- nature
- wellbeing
- "Reconnect with nature, self & others"
- intuition
- "ancient tools"
- "transformational experiences"
- "learn, heal and grow"
- "personal, social and environmental change"
- "outer change in the world starts with the self"

## Center for Systems Awareness

- Profile: https://ecosystem.lifeitself.org/tbd/Center%20for%20Systems%20Awareness
- Markdown file (metadata fields, such as location, year founded and team size, are visible): https://github.com/life-itself/ecosystem/blob/main/content/tbd/Center%20for%20Systems%20Awareness.md

![Screenshot 2023-12-20 at 09 21 56](https://github.com/life-itself/community/assets/97066487/d0783789-f7c3-4086-920a-3972ccdcd76c)

### Keywords

- "systemic well-being at all levels, from the individual to the larger systems of institutions, society, economy and ecology"
- "compassionate systems"
- systems awareness
- "sustaining deep change over time"
- renaissance
- radical; bold
- "profound inter-connectedness as the integrating idea – that 'self,' 'other,' society and the larger natural world all arise as entwined systems of relationships"
- "recovering our sense of connectedness and caring"

## Joysong Collective

- Profile: https://ecosystem.lifeitself.org/tbd/Joysong%20Collective
- Markdown file (metadata fields, such as location, year founded and team size, are visible): https://github.com/life-itself/ecosystem/blob/main/content/tbd/Joysong%20Collective.md

![Screenshot 2023-12-20 at 09 22 07](https://github.com/life-itself/community/assets/97066487/091d590b-e158-4394-95cb-aea829bfcfa6)

### Keywords

- aliveness
- joy
- alignment
- spirit
- "felt sense"
- "listen deeply"
- "new/ancient"
- "deep yearning for full-throated aliveness"
- "we simply know in our bones that we can be more fully alive in ways our current paradigms don’t support"
- "re-discover our fundamental joy"
- "remembering our already-wholeness"
- regeneration
- sangha, dana (Buddhist influence)

#### 1) *Re*connecting to what we already know
42 Acres speaks of "reconnect[ing] with nature, self & others", while the Center for Systems Awareness speaks of the need to "recove[r]our sense of connectedness and caring" at multiple levels, including "self, other, society and the larger natural world". Joysong Collective speak of "re-discover[ing] our fundamental joy" and "remembering our already-wholeness".

*Re*connect, *re*cover, *re*-discover, and *re*member suggest finding a way back to 'true' human nature, and to qualities/awarenesses/ways of being that are already innate and known, albeit in some ways lost, forgotten, or obscured.

#### 2) Bold and radical innovation/experimentation
Center for Systems Awareness use words like "bold new ideas", "radical", and "renaissance". 42 Acres speak of curating "transformational experiences". At the same time as there is a sense of return to something essential (see point above), there is also a flavour of innovation and newness. 

Joysong Collective encapsulates this juxtaposition when they talk of "experimenting with new/ancient ... tools."

#### 3) Engagement with concrete issues
The Center for Systems Awareness argue that the cultivation of awareness of wholeness, connectedness, and interdependence must take place in the midst of engagement with real-world, current problems: that we need people who are "equally engaged in the world and in their own cultivation – as two facets of the same journey".

What does it mean to be engaged in the world? 42 Acres demonstrate this in their focus on environmental change (alongside personal and social change), particularly in their regenerative and hyper-local approach to food and farming at their retreat centre. An example of the Center for Systems Awareness's engagement in concrete issues is their Refugee Education project: https://systemsawareness.org/project/transforming-refugee-education-towards-excellence/. As part of Joysong Collective's "Endowment" activity, "50% of funds will be allocated into local projects" -- although it is not clear what kinds of projects these will be.

### Differences:

- In what ways are they distinct from each other?

## Principles of our mapping approach

- Wiki over database
  - Getting a grasp of an organisation is priority over categorisation (that can come later). I.e. focus on key info over pre-determined fields to fill in.
  - Use links for categorization if you want (à la wikipedia)
- Focus on quality of profiles in terms of summary and drawing out the things interesting to us ...
  - e.g. their philosophy/theory of change
  - e.g. what they *actually* do
  - e.g. "impact" or potential impact - whether they are "getting stuff done" and doing so in sustainable way (?)
- Then do discussion to clarify things and contribute to sensemaking 
  - e.g. what tribes are emerging
  - what features do we care about ...

**Comment by matthewmccarthy11** [comment]
It would be interesting to eventually outline the interesting threads/tribes that each org is apart of, and maybe make that as part of the profile- ie. what tribe is it apart of?- but I think that would take more time, and come only once we have identified the tribes/ threads more.

**Comment by rufuspollock** [comment]
## What works

Really great to have well-written up exemplars.

### Simple, well selected quotes to address particular items

e.g. https://ecosystem.lifeitself.org/tbd/42-Acres and https://ecosystem.lifeitself.org/tbd/Center%20for%20Systems%20Awareness

![image](https://github.com/life-itself/community/assets/180658/1a13a847-38e9-48e2-ba7f-ea4ef688f9bf)

![image](https://github.com/life-itself/community/assets/180658/8a2970c6-f7ac-435a-b865-f92b6b3e756a)

### Key people is useful

e.g. on 42 acres.

🚩 Note it is missing on https://ecosystem.lifeitself.org/tbd/Center%20for%20Systems%20Awareness - should have Peter Senge (the co-founder)

---
---

## Questions / comments

### Connections section seems a bit ambiguous ...

- What are connections e.g. We will dance with mountains at bottom of Joysong Collective?
- https://ecosystem.lifeitself.org/tbd/Center%20for%20Systems%20Awareness has a whole bunch of connections but more like inspirations.

![image](https://github.com/life-itself/community/assets/180658/8245abe0-6f1f-4633-ad85-88851fc22ea0)

**Suggestion:** connections should be at bottom of page and we can start breaking into sections e.g. "Inspirations", "Collaborators" or whatever ...

### What cultural paradigm(s) are they advocating for ...

- Once you start to look many are in "green/post-modern"
- This could be a useful distinction

### What about retreat centers

- 42 acres seems like an updated retreat center for the 21st century. what makes it different? 
  - If it isn't that different does that mean most retreat centers are part of PIE?

**Reply by catherinet1** [reply]
Yes, agreed re moving connections to bottom of the page and using sub-headings, e.g. Partners, Inspirations etc.

**Comment by matthewmccarthy11** [comment]
Interesting to consider 'post modern vocabulary' and 'meta modern vocabulary': 'we discussed that 'regenerative' or 'well-being' can at times be more post modern than meta modern, as an example. Maybe 'post modern vocabulary' is more concerned with 'advertising', but also worth emphasizing that all orgs try to attract a certain audience. 

What would meta modern vocabulary be, if there is one? Perhaps renaissance, radical, evolution, conscious cultural evolution.

Both post modern and meta modern **might** have a particular 'framing'

Part of the problem with teal is that we like to grasp things intellectually: how do we embody integration? How would a teal organization work? 

Matthew: the language of 'goals' and 'symbiosis' might be interesting here, in the sense that 'goals' (for example the goals of a post modern organization, like financial stability, spectacle) emerge out of the 'goals' in a 'post modern world'. Meta modern organizations seem to have 'goals' which don't emerge out of the goals of a 'post modern world' but emerge out of 'meta modern' goals, which are integration, stability.

There is a sense in which this is an opportunity for 'mapping the evolution of the paradigm' from Post modern to Meta modern. In a way, it makes a lot of sense that there is 'social conflict' in post modernism, since there is multiple perspectives, and a start of an integration of worldviews.

Potential post modern tendencies: to be 'generous' in how they estimate their own efforts

**Reply by asimong** [reply]
@matthewmccarthy11 maybe you'd like to add here a link to the new one where you're carrying on the conversation? Or I can, here: #973
